core/templates/dev/head/pages/exploration_editor/translation_tab/TranslationTabDirective.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the translation tab.
+ */
+
+oppia.directive('translationTab', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {},
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/exploration_editor/translation_tab/' +
+        'translation_tab_directive.html'),
+      controller: [
+        '$scope', function($scope) {
+          $scope.underConstruction = true;
+          $scope.$on('refreshAudioTab', $scope.refreshSettingsTab);
```

Comment:
Where is `refreshAudioTab` broadcasted from?

Reply:
Done!